recipe/meta.yaml

```diff
@@ -20,6 +20,61 @@ source:
 build:
   number: 0
   skip: True               # [py<37]
+  entry_points:
```

Comment:
just out of curiosity how did know about those entry points ?

Reply:
Bad addition. I removed them.